src/components/Settings/TwoStepFaq.vue

```diff
@@ -1,45 +1,41 @@
 <template>
-	<div>
-		<kv-settings-card>
```

Comment:
@BoulderBrains I pulled the content out of KvSettingsCard since this section didn't have an icon or the same header styles anymore.